README.md

```diff
@@ -1,92 +1,150 @@
-
-Those need to be manually extracted, and copied to C:\Program Files (x86)\NSIS\Plugins
-
-Specifically, the following files need to exist:
-
-    C:\Program Files (x86)\NSIS\Plugins\LockedList64.dll
-    C:\Program Files (x86)\NSIS\Plugins\Registry.dll
-    C:\Program Files (x86)\NSIS\Plugins\x86-unicode\LockedList.dll
-    C:\Program Files (x86)\NSIS\Plugins\x86-unicode\Registry.dll
-    C:\Program Files (x86)\NSIS\Plugins\x86-unicode\INetC.dll
-
-
-### Building all dependencies from source
-
-This guide provides a "download.bat" and "build.bat" file to automatically build all dependencies.
-
-Start "x64 Native Tools Command Prompt for VS 2022"
-
-Run download.bat
-
-This should download all necessary sources.
-
-
-#### Build for debug
-
-    build.bat debug
-
-
-#### Build for release
-
-    build.bat release
-
-
-### Building strawberry
-
-Add "c:\strawberry_msvc_x86_64_debug\bin" for the debug version, or "c:\strawberry_msvc_x86_64_release\bin" for the release version to PATH.
-
-Open the strawberry git source directory in Visual Studio 2022.
-
-Add the following extra CMake arguments:
-
-For debug:
-
-    -DARCH=x86_64 -DICU_ROOT=c:\\strawberry_msvc_x86_64_debug
-
-
-For release:
-
-    -DARCH=x86_64 -DICU_ROOT=c:\\strawberry_msvc_x86_64_release
+## Links
+
+- [Strawberry Music Player Website](https://www.strawberrymusicplayer.org)
+- [Wiki](https://wiki.strawberrymusicplayer.org/wiki/Main_Page)
+- [Forum](https://forum.strawberrymusicplayer.org)
+- [Github sources](https://github.com/strawberrymusicplayer/strawberry)
+- [Github MSVC](https://github.com/strawberrymusicplayer/strawberry-msvc)
+- [Github MSVC prebuilt binary dependencies](https://github.com/strawberrymusicplayer/strawberry-msvc-dependencies)
+
+## Manual installation on Windows
+
+Download and install manually:
+- [Git](https://git-scm.com/downloads)
+    - Default settings can be used
+- [Visual Studio 2022 Community](https://visualstudio.microsoft.com/vs/)
+    - Select `Desktop development with C++`
+
+## Windows setup
+
+Open a Windows Terminal and type:
+
+```
+mkdir C:\data\projects\strawberry
+cd C:\data\projects\strawberry
+git clone https://github.com/strawberrymusicplayer/strawberry-msvc.git
+cd strawberry-msvc
+.\download.bat
+```
+
+Install manually from `C:\data\projects\strawberry\msvc_\downloads`:
+- **Required**:
+    - Install `7z<VERSION>-x64.exe`
+    - Install `cmake-<VERSION>-windows-x86_64.msi`
+    - Install `python-<VERSION>-amd64.exe`
+        - In first dialog:
+            - Check: `Add python.exe to PATH`
+            - Click `Install Now` (to use defaults)
+        - In dialog `Setup was successful`:
+            - Click: `Disable path length limit`
+    - Install `strawberry-perl-<VERSION>-64bit.msi`
+    - Extract `win_flex_bison-<VERSION>.zip`
+- **Recommended: When building dependencies from source**:
+    - Install `nasm-<VERSION>-installer-x64.exe`
+        - Run as normal user and click OK 
```

Comment:
I suggest to change this to a normal installation using administrator to ``C:\Program Files\NASM``

Reply:
Correct, path `C:\Program Files\NASM` is used in `build.bat`, so there is no environment variable needed.

Fixed in 83adf8f6c50e2a7f070a760bc17b2987531e0229.